src/ImageRunModal.scss

```diff
@@ -0,0 +1,18 @@
+@import '@patternfly/patternfly/layouts/Grid/grid.css';
+
+// Limit the max-width more the override in podman.scss which is 40%
+#run-image-dialog-memory, #run-image-cpu-priority {
+     max-width: 15%;
+}
+
+#memory-unit-select {
+     max-width: 20%;
+}
+
+#run-image-dialog-memory-limit-checkbox, #run-image-dialog-cpu-priority-checkbox {
+     margin-top: 12px;
+}
+
+.pf-c-form__group-label {
```

Comment:
When you override pf rules please use a specific className in addition to the PF4 class. In this way you can prevent this from getting applied to all other random places.

Reply:
:100: 

This, exactly. I totally agree with @KKoukiou. 

We've hit bugs before due to this. It's often fine to do `.my-item .pf-c-thing`, if you have complete control of the item and it's specific to your page. But if you're doing a 1-off in your page, it's usually better to have a custom class. We usually use `.ct-` as a prefix for reusable components specific to Cockpit and I like to use something unique to the page for other things, just to make it clear what the item applies to. 

But if you can, use a PF4 utility class when it's what you want.

Reply:
Note, since this is in the ImageRunModal.scss, you might assume it's only going to apply to the image run modal. It won't. It will apply to the whole page where it's imported. (Other people have made that wrong assumption before.) This is why it's important to use custom class names or to scope the CSS to a particular part of the page using CSS itself.

Reply:
It was actually not required by anything and dropped.